### ISSUE TITLE
add chainmail recipe from the existing iron mesh

### ISSUE THREAD

**Issue by Efimero** [An-Sar/PrimalCore#48]
is it already planned? seems like a logical step, but it's good to have it on the tracker just so it's not forgotten =3

**Comment by An-Sar** [comment]
There will likely be a filler recipe in Primal that gets superseded when/if Kitsu's ForgeCraft is installed. He is planning on doing a complete armor crafting process.

**Comment by Efimero** [comment]
yeah, cool, but there's no point having that mode when you want to be able to craft neat mod items =P
however Primal works very well with other mods that have high tier stuff, so the chainmail recipe would be a good thing to have to get better armor for less iron
